packages/plugin-autocapture-browser/src/frustration-plugin.ts

```diff
@@ -24,7 +24,16 @@ import { createClickObservable, createMutationObservable } from './observables';
 
 type BrowserEnrichmentPlugin = EnrichmentPlugin<BrowserClient, BrowserConfig>;
 
-export const frustrationPlugin = (options: FrustrationInteractionsOptions): BrowserEnrichmentPlugin => {
+export const frustrationPlugin = (
+  options: FrustrationInteractionsOptions = {
+    deadClicks: {
+      cssSelectorAllowlist: DEFAULT_DEAD_CLICK_ALLOWLIST,
+    },
+    rageClicks: {
+      cssSelectorAllowlist: DEFAULT_RAGE_CLICK_ALLOWLIST,
+    },
+  },
+): BrowserEnrichmentPlugin => {
```

Comment:
What's the point of adding a partial default value here if we do the following a few lines below? 
```
const rageCssSelectors = options.rageClicks?.cssSelectorAllowlist ?? DEFAULT_RAGE_CLICK_ALLOWLIST;
  const deadCssSelectors = options.deadClicks?.cssSelectorAllowlist ?? DEFAULT_DEAD_CLICK_ALLOWLIST;
  const dataAttributePrefix = options.dataAttributePrefix ?? DEFAULT_DATA_ATTRIBUTE_PREFIX;
```

This blocks brings in confusion, for example, if i pass custom a css selector allow list for dead clicks, do i loose the default css selector allow list for rage click?